### PR TITLE
Update for tf 13

### PR DESCRIPTION
**Pull request by ebalduf**
These fixes accomplish 2 things. 1) add 4 new Google regions 2) allow this module to work with Terraform 13. 
Tests will not run properly until image gcr.io/cloud-foundation-cicd/cft/developer-tools gets an update to kitchen for TF 13.  I the mean time, I have hand rolled a manual test to confirm this module.

### CODE REVIEW

**Comment by morgante** [comment]
@ebalduf Could you split this into two PRs (1 for regions, 1 for TF 0.13)?

**Comment by ebalduf** [comment]
I split out the region request.  I will change this one to be the TF 13 update only.

File: main.tf [hunk 1]
```diff
@@ -15,7 +15,7 @@
  */
 
 terraform {
-  required_version = ">= 0.12.0"
+  required_version = ">= 0.12.0, < 0.14.0"
```
Comment (morgante) [inline main.tf]: Could we remove this block entirely? It's redundant with `versions.tf`?

**Review by morgante — CHANGES_REQUESTED** [review]

**Review by morgante — APPROVED** [review]
Thanks!

**Pull request merged**